gradle/dependencies.gradle

```diff
@@ -15,367 +15,386 @@
+libraries.commons_cli =         'commons-cli:commons-cli'
+libraries.commons_codec =       'commons-codec:commons-codec'
+libraries.commons_collections = 'commons-collections:commons-collections'
+libraries.commons_compress =    'org.apache.commons:commons-compress'
+libraries.commons_httpclient =  'org.apache.httpcomponents:httpclient'
+libraries.commons_io =          'commons-io:commons-io'
+libraries.commons_lang =        'commons-lang:commons-lang'
+libraries.fastutil =            'it.unimi.dsi:fastutil'
+libraries.gcs =                 'com.google.apis:google-api-services-storage'
+libraries.groovy =              'org.codehaus.groovy:groovy-all'
+libraries.gson =                'com.google.code.gson:gson'
+libraries.guava =               'com.google.guava:guava-jdk5'
+libraries.inject =              'javax.inject:javax.inject'
+libraries.ivy =                 'org.apache.ivy:ivy'
+libraries.jackson_core =        'com.fasterxml.jackson.core:jackson-core'
+libraries.jackson_annotations = 'com.fasterxml.jackson.core:jackson-annotations'
+libraries.jackson_databind =    'com.fasterxml.jackson.core:jackson-databind'
+libraries.jansi =               'org.fusesource.jansi:jansi'
+libraries.jatl =                'com.googlecode.jatl:jatl'
+libraries.jcifs =               'org.samba.jcifs:jcifs'
+libraries.jcip =                'net.jcip:jcip-annotations'
+libraries.jgit =                'org.eclipse.jgit:org.eclipse.jgit'
+libraries.joda =                'joda-time:joda-time'
+libraries.jsch =                'com.jcraft:jsch'
+libraries.jsr305 =              'com.google.code.findbugs:jsr305'
+libraries.junit =               'junit:junit'
+libraries.kryo =                'com.esotericsoftware.kryo:kryo'
+libraries.maven3 =              'org.apache.maven:maven-core'
+libraries.maven3_wagon_file =   'org.apache.maven.wagon:wagon-file'
+libraries.maven3_wagon_http =   'org.apache.maven.wagon:wagon-http'
+libraries.nativePlatform =      'net.rubygrapefruit:native-platform'
+libraries.nekohtml =            'net.sourceforge.nekohtml:nekohtml'
+libraries.objenesis =           'org.objenesis:objenesis'
+libraries.plexus_container =    'org.codehaus.plexus:plexus-container-default'
+libraries.plist =               'com.googlecode.plist:dd-plist' // for XCode IDE integration support
+libraries.pmaven_common =       'org.sonatype.pmaven:pmaven-common'
+libraries.pmaven_groovy =       'org.sonatype.pmaven:pmaven-groovy'
+libraries.rhino =               'org.mozilla:rhino'
+libraries.simple =              'org.simpleframework:simple'
+libraries.testng =              'org.testng:testng'
+libraries.xerces =              'xerces:xercesImpl'
+libraries.xmlApis =             'xml-apis:xml-apis'
+libraries.slf4j_api =           'org.slf4j:slf4j-api'
+libraries.jcl_to_slf4j =        'org.slf4j:jcl-over-slf4j'
+libraries.jul_to_slf4j =        'org.slf4j:jul-to-slf4j'
+libraries.log4j_to_slf4j =      'org.slf4j:log4j-over-slf4j'
```

Comment:
I'd be tempted to get rid of all of these and just use the module id directly wherever it's required. I'm not sure giving our own alias is beneficial. (But I'm OK to leave it like this for now)